packages/strapi-design-system/src/MainNav/NavBrand.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '../Box';
+import { Text, TextButton } from '../Text';
+import { Row } from '../Row';
+import { useMainNav } from './MainNavContext';
+import { VisuallyHidden } from '../VisuallyHidden';
+
+const BrandIconWrapper = styled.div`
```

Comment:
Shouldn't it be a link as well?

Reply:
I've asked Maeva and she answered no :). Maybe it will change in the coming months 😄

Reply:
As there is no link to go back to the home page, I think it will change also.